/**:
  ros__parameters:
    robot_description_kinematics:
      ur_manipulator:
        kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
        kinematics_solver_timeout: 0.05
        kinematics_solver_attempts: 3
        kinematics_solver_search_resolution: 0.005
    robot_description_semantic: |
      <?xml version="1.0" ?>
      <robot name="ur10e">
        <virtual_joint name="virtual_joint" type="fixed" parent_frame="world" child_link="base_link"/>
        <group name="ur_manipulator">
          <chain base_link="base_link" tip_link="tool0"/>
        </group>
        <group_state name="home" group="ur_manipulator">
          <joint name="shoulder_pan_joint" value="0"/>
          <joint name="shoulder_lift_joint" value="-1.57"/>
          <joint name="elbow_joint" value="1.57"/>
          <joint name="wrist_1_joint" value="0"/>
          <joint name="wrist_2_joint" value="1.57"/>
          <joint name="wrist_3_joint" value="0"/>
        </group_state>
      </robot>
